ts/parts-more/Polar.ts

```diff
@@ -685,81 +707,52 @@ if (seriesTypes.column) {
                                 // range, set it to 0
                                 if ((start as any) < 0) {
                                     start = 0;
+                                } else if ((start as any) > visibleRange) {
+                                    start = visibleRange;
                                 }
                             }
                         }
                     } else {
-                        // Calculating starting and ending angles for
-                        // inverted polar bars
-                        start = 0;
+                        // Initial start and end angles for radial bar
+                        start = thresholdAngleRad;
                         end = point.plotY;
                     }
 
-                    // Prevent from rendering point outside acceptable
-                    // circle range
                     if ((start as any) > (end as any)) {
```

Comment:
We should avoid casting `as any`

Reply:
Previously mentioned check on the thresholdAngleRad property and checking the start variable resolves this problem as well.